modules/raft/src/main/java/org/apache/ignite/internal/raft/RaftGroupServiceImpl.java

```diff
@@ -468,6 +468,7 @@ public CompletableFuture<Long> readIndex() {
         Function<Peer, ? extends NetworkMessage> requestFactory = p -> factory.readIndexRequest()
                 .groupId(groupId)
                 .peerId(p.consistentId())
+                .serverId(p.consistentId())
```

Comment:
It looks strange that there is no failing test. Let's add a test, where we can check that read index request is working in the happy pass scenario.

Reply:
added `ItJraftCounterServerTest#testReadIndex`